sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples/Sample_HealthcareAsync_Pagination.cs

```diff
@@ -27,12 +27,17 @@ public async Task HealthcareAsyncPagination()
 
             var list = new List<string>();
 
-            for (int i = 0; i < 23; i++)
+            for (int i = 0; i < 10; i++)
             {
                 list.Add(document);
             };
 
-            HealthcareOperation healthOperation = await client.StartHealthcareBatchAsync(list);
+            HealthcareOptions options = new HealthcareOptions()
+            {
+                Top = 2
+            };
+
+            HealthcareOperation healthOperation = await client.StartHealthcareBatchAsync(list, "en", options);
```

Comment:
Not that you need to change it, but another thing you could do to pass options and not language is
`HealthcareOperation healthOperation = await client.StartHealthcareBatchAsync(list, options:options);`